util/omrutil/j9memclr.cpp

```diff
@@ -44,14 +46,18 @@ struct IHAPSA {
 };
 #endif
 
-#if defined (S390)
-static int isZ10orGreater = -1;
-#endif
+#if defined(J9ZOS390) || (defined(LINUX) && defined(S390))
+#include <stdlib.h>
```

Comment:
Sorry, just seeing this now. This include should be at the top of the file with the rest of them (system includes should be grouped together, and includes are sorted alphabetically, so this should be right before `string.h`).